packages/util/src/combine-fns.ts

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
So are you moving this into util from somewhere? 

This is *in addition* to TS conversion, right?

Reply:
no this was in util before. my git mv, commit, change, commit process doesn't work on diff for some reason